src/lib/GamePage/Accordion.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+	import { slide } from 'svelte/transition';
+	export let variation;
+	let isOpen = false;
+	const toggle = () => (isOpen = !isOpen);
+</script>
+
+<button on:click={toggle} aria-expanded={isOpen}> {variation.description} </button>
+
+{#each variation.actions as action}
+	{#if isOpen}
+		<ul transition:slide={{ duration: 300 }}>
+			{#each variation.actions as action}
+				{action.description || action}
+			{/each}
+		</ul>
+	{/if}
+{/each}
+
+<style>
+	button {
+		background: transparent;
+		border: 2px solid var(--color-light-orange);
+		width: 100%;
+		padding: 0.7rem 1rem;
```

Comment:
```suggestion
		padding: 0.7em 1em;
```
Automatisme zeker, maar hadden volgens mij afgesproken om geen `rem` meer te doen in dit project.

Reply:
ik gebruik zowel em als rem, want ze hebben beide een use case

Reply:
dat is wat we afgesproken hadden (dacht ik?)

Reply:
Ja volgens mij ook